src/rsa.rs

```diff
@@ -27,6 +28,30 @@ pub struct RSAPKCS1Padding {
     digestinfo_prefix: &'static [u8],
 }
 
+impl RSAPKCS1Padding {
+    fn pad(&self, msg: &[u8], out: &mut [u8]) -> Result<(), ()> {
+        let digest_len = self.digestinfo_prefix.len() + self.digest_alg.output_len;
+        if out.len() < digest_len + 11 {
```

Comment:
Please document where 11 comes from with a comment. Also, please write `digest_len + 11` as (`1 + 1 + 8 + digest_len`) to make things even more clear.